lib/msal-react/README.md

```diff
@@ -86,6 +86,8 @@ npm run test:coverage
 
 For help getting started with `@azure/msal-react` please see our [getting started](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/getting-started.md) doc.
 
+Migrating from [react-aad-msal](https://www.npmjs.com/package/react-aad-msal)? Check out our [migration guide](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/migration-guide.md).
+
 ### MSAL Basics
 
 Since `@azure/msal-react` is built on top of `@azure/msal-browser` a lot of the same docs are relevant. For concepts specific to `@azure/msal-react` please see [below](#msal-react-specific-concepts)
```

Comment:
```suggestion
Since `@azure/msal-react` is built as a wrapper for `@azure/msal-browser` many docs from the `msal-browser` repo are relevant. For concepts specific to `@azure/msal-react` please see [below](#msal-react-specific-concepts)
```